pptx/text/text.py

```diff
@@ -339,6 +339,32 @@ def italic(self):
     def italic(self, value):
         self._rPr.i = value
 
+    @property
```

Comment:
These look handy but unrelated to the `duplicate_slide()` method... are they just some utility stuff for the future or do they have a specific purpose?

Reply:
Utility that I need to fru's building super and subscript for things like 2nd, 3rd, 4th, etc.